pkg/apply/apply.go

```diff
@@ -0,0 +1,65 @@
+package apply
```

Comment:
@bcrochet  the plan to reuse the code from CNO and to drop this directory, right?